files/fr/glossary/code_splitting/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Code splitting
+slug: Glossary/Code_splitting
+tags:
+  - Glossary
+  - Reference
+  - Web Performance
+  - code splitting
+  - latency
+translation_of: Glossary/Code_splitting
+original_slug: Glossaire/Code_splitting
+---
+<p><span class="seoSummary"><strong>Le code splitting</strong> est le fractionnement du code en divers bundles ou composants qui peuvent ensuite être chargés à la demande ou en parallèle.</span></p>
```

Comment:
The `seoSummary` is no longer used so it would be nice to remove this `span` entirely :)